### PR TITLE
fix(django): add error when trying to insert binary data

### PR DESCRIPTION
**Pull request by taylor-cedar**
## Summary
This library doesn't support uploading binary data. It incorrectly serializes the data and results in invalid data being inserted. This can eventually be fixed by using psycopg 3 with Django 4 and using the binary copy instead of the CSV format. 

For now, just error so we don't corrupt data

##### How can reviewers test your change manually?
Check that it throws an error on binary data
##### What automated tests did you add? If none, please state why.
Added test for error

## Security Impact
Please answer the questions below about your PR. 
If yes, please explain and add `cedar-team/security` as reviewers.

##### Add/modify authentication, authorization, encryption, or HTTP headers?
No
##### Add/modify any PII or PHI data fields?
No
##### Add new third parties the app interacts with (e.g., Nordis/Stripe)?
No
##### Ask patients to enter a new kind of data, (e.g. insurance capture)?
No
##### Impact any of the other [Security Engineering Practices](https://careportal.atlassian.net/wiki/spaces/SEC/pages/1192690295/Security+Engineering+Practices)?
_Protect sensitive data, Validate your inputs, Know your user, Establish trust boundaries, Keep good records, Expect vulnerabilities, Follow least privilege_

No

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -6,8 +6,14 @@ services:
...
-      - db
+      db:
+        condition: service_healthy
   db:
     image: postgres:10
+    healthcheck:
+      test: [ "CMD-SHELL", "pg_isready -U postgres" ]
+      interval: 5s
+      timeout: 5s
+      retries: 5
```
Comment (sbernstein-cedar) [inline docker-compose.yml:17]: I'm not really familiar with these files but I assume this is fine?

**Review by sbernstein-cedar — APPROVED** [review]
- [x] I fully understand what the code is doing and its context
- [x] The code is the best way to accomplish this problem (I would code it similarly)
- [x] The code is well organized, easy to understand and matches Cedar design patterns and best practices
- [x] The code handles edge cases (that I can think of) and has tests to validate them
- [x] The appropriate people are reviewing (knowledge and security)
- [ ] I tested the code personally (Not required)

File: docker-compose.yml (hunk 1 above)
Comment (taylor-cedar) [reply docker-compose.yml:17]: Yes, this is to just check postgres is ready before running the tests
